From 4db5178bb6682f0edca13d2c69a23548a46efefe Mon Sep 17 00:00:00 2001
From: Olivier Crete <olivier.crete@collabora.com>
Date: Thu, 26 Apr 2018 17:19:06 -0400
Subject: [PATCH 2/2] libtool: Stop adding -bind_at_load on all Darwin

It was only needed on antique macOS that we don't support
and it breaks -fembed-bitcode.
---
 build-aux/ltmain.sh | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/build-aux/ltmain.sh b/build-aux/ltmain.sh
index c5b34b3..749eda9 100644
--- a/build-aux/ltmain.sh
+++ b/build-aux/ltmain.sh
@@ -10227,16 +10227,6 @@ EOF
 
       case $host in
       *-*-darwin*)
-	# Don't allow lazy linking, it breaks C++ global constructors
-	# But is supposedly fixed on 10.4 or later (yay!).
-	if test CXX = "$tagname"; then
-	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
-	    10.[0123])
-	      func_append compile_command " $wl-bind_at_load"
-	      func_append finalize_command " $wl-bind_at_load"
-	    ;;
-	  esac
-	fi
 	# Time to change all our "foo.ltframework" stuff back to "-framework foo"
 	compile_deplibs=`$ECHO " $compile_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g' | $SED 's% \([^ $]*\).ltweak_framework% -weak_framework \1%g'`
 	finalize_deplibs=`$ECHO " $finalize_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g' | $SED 's% \([^ $]*\).ltweak_framework% -weak_framework \1%g'`
-- 
2.15.1 (Apple Git-101)

